kratos/mpi/includes/mpi_coloring_utilities.h

```diff
@@ -0,0 +1,259 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+
+#if !defined(KRATOS_MPI_COLORING_UTILITIES_H_INCLUDED )
+#define  KRATOS_MPI_COLORING_UTILITIES_H_INCLUDED
+
+// System includes
+#include <iostream>
+#include <sstream>
+#include <map>
+
+// External includes
+
+
+// Project includes
+#include "includes/define.h"
+#include "includes/serializer.h"
+
+
+namespace Kratos
+{
+///@addtogroup ApplicationNameApplication
+///@{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/// Short class definition.
+/** Detail class definition.
+*/
+class MPIColoringUtilities
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of MPIColoringUtilities
+    KRATOS_CLASS_POINTER_DEFINITION(MPIColoringUtilities);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Default constructor.
+    MPIColoringUtilities() {}
+
+    /// Destructor.
+    virtual ~MPIColoringUtilities() {}
+
+    ///@}
+    ///@name Operators
+    ///@{
+    std::vector<int> ComputeRecvList(
+        const std::vector<int>& destination_ids,
```

Comment:
Please try to follow the style guide for function arguments
```suggestion
        const std::vector<int>& rDestinationIds,
```